R/ml_clustering_bisecting_kmeans.R

```diff
@@ -147,7 +147,16 @@ new_ml_bisecting_kmeans_model <- function(jobj) {
 }
 
 new_ml_bisecting_kmeans_summary <- function(jobj) {
-  new_ml_clustering_summary(
+  sc <- spark_connection(jobj)
+  version <- spark_version(sc)
+
+  bisecting_kmeans_summary <- new_ml_clustering_summary(
     jobj,
     class = "ml_bisecting_kmeans_summary")
+
+  if (version >= "3.0.0") {
```

Comment:
Since you added `is_spark_3`, we could expand this function to support connections and `jobs` with something like:

```
is_spark_3 <- function(x) {
  UseMethod(""is_spark_3)
}

is_spark_3.spark_connection <- function(x) {
  version <- spark_version(x)
  version >= "3.0.0"
}

is_spark_3.spark_jobj <- function(x) {
  sc <- spark_connection(x)
  is_spark_3(sc)
}
```

Reply:
This is good, not know that.

Reply:
Here is more about S3 (one of various class systems available in R): http://adv-r.had.co.nz/S3.html